### PR TITLE
add Virtual dry static energy

### PR DESCRIPTION
**Pull request by yairchn**
Add dry static energy

### CODE REVIEW

**Comment by szy21** [comment]
Should we add some tests for this?

File: src/relations.jl [hunk 1]
```diff
@@ -2714,6 +2715,24 @@ function moist_static_energy(
     return specific_enthalpy(param_set, ts) + e_pot
 end
 
+"""
+    virtual_dry_static_energy(param_set, ts, e_pot)
+
+Virtual dry static energy, given
+ - `param_set` an `AbstractParameterSet`, see the [`Thermodynamics`](@ref) for more details
+ - `ts` a thermodynamic state
+ - `e_pot` potential energy (e.g., gravitational) per unit mass
```
Comment (tapios) [inline src/relations.jl:2724]:
```suggestion
 - `e_pot` gravitational potential energy per unit mass
```
(Don't we call this Phi elsewhere?)

**Review by charleskawczynski — APPROVED** [review]

**Comment by szy21** [comment]
@charleskawczynski What did I do wrong on the doc string?

File: src/relations.jl (hunk 1 above)
Comment (szy21) [reply src/relations.jl:2724]: I'm not very familiar with the thermodynamics package, but I'm following e.g. moist static energy [here](https://github.com/CliMA/Thermodynamics.jl/blob/9a4bd9d145cbfff879f6768e737276ee47951750/src/relations.jl#L2709). Maybe we call this Phi somewhere in ClimaAtmos? In that case we can make it consistent in ClimaAtmos later.

Comment (tapios) [reply src/relations.jl:2724]: e_pot is fine if that is what we do in Thermodynamics.

**Pull request merged**